README.md

```diff
@@ -10,7 +10,16 @@ Crystal is a programming language with the following goals:
 * Have compile-time evaluation and generation of code, to avoid boilerplate code.
 * Compile to efficient native code.
 
-## Build
+## Contributing to These Docs
```

Comment:
Is it clear that by "crystal book" we are talking about what you're looking at though? On the crystal-lang.org homepage they're just referred to as "docs" and "api".